src/stabilities/types.ts

```diff
@@ -0,0 +1,46 @@
+type Price = {
+  title: string;
+  display: string;
+}
+
+type Inflation = {
+  title: string;
+  display: string;
+}
+
+type MarketCap = {
+  title: string;
+  display: string;
+}
+
+type CommunityPool = {
+  title: string;
+  display: string;
+}
+
+type Data = {
+  total: Total;
+  detail: Detail[];
```

Comment:
Because of how you're using the detail you need to be more specific
```
type Data = {
  total: Total;
  detail: [Detail, Detail];
}
```
This tells use we expect 2 and only 2 `Detail` items in the array. Nothing more nothing less

Reply:
got it